firefox-ios/Client/Frontend/PasswordGenerator/PasswordGeneratorMiddleware.swift

```diff
@@ -109,4 +118,26 @@ final class PasswordGeneratorMiddleware {
     private func clearGeneratedPasswordForSite(origin: String, windowUUID: WindowUUID) {
         generatedPasswordStorage.deletePasswordForOrigin(origin: origin)
     }
+
+    private static func loadPasswordRules() async -> [PasswordRuleRecord]? {
```

Comment:
I might be missing the reason for this, but how come we're making this method static?

Reply:
It doesn't need to be an instance method 😄